File: relation/association/aggregation/seller.kt
class Shop(
    private val id: String,
    private val name: String
) {
    var seller: Seller

        set(value) {
            seller = value
        }
        get() {
            return seller
        }

}

class Seller(
    private val id: String,
    private val name: String
)

File: relation/association/aggregation/seller.dart
class Shop {
  String _id;
  String _name;
  Seller _seller;

  Shop(this._id, this._name);

  Seller get seller => _seller;

  set seller(Seller value) {
    _seller = value;
  }
}

class Seller {
  String _id;
  String _name;
}
